Sources/Public/CalendarView.swift

```diff
@@ -879,6 +903,57 @@ public final class CalendarView: UIView {
     }
   }
 
+  private func configureMultipleDaySelectionPanGestureRecognizer() {
+    if multipleDaySelectionDragHandler == nil {
+      removeGestureRecognizer(multipleDaySelectionGestureRecognizer)
+    } else {
+      addGestureRecognizer(multipleDaySelectionGestureRecognizer)
+    }
+  }
+
+  @objc
+  private func multipleDaySelectionGestureRecognized(
+    _ gestureRecognizer: UILongPressGestureRecognizer)
+  {
+    guard gestureRecognizer.state != .possible else {
+      lastMultipleDaySelectionDay = nil
+      return
+    }
+
+    let locationInScrollView = gestureRecognizer.location(in: scrollView)
+
+    var intersectedDay: Day?
+    for subview in scrollView.subviews {
+      guard
+        let itemView = subview as? ItemView,
+        case .layoutItemType(.day(let day)) = itemView.itemType,
+        itemView.frame.contains(locationInScrollView)
```

Comment:
```suggestion
                itemView.frame.contains(locationInScrollView)
```

Reply:
hmm, I do this in a few other places in this file. However, I don't do this:

```
else {
  continue
}
```

and instead was doing this:
```
else
{
  continue
}
```

Not sure I like this anymore but I'll update so it's consistent, then do a refactor all at once if I want to change it.